js/app.js

```diff
@@ -0,0 +1,13 @@
+'Use strict'
+
+
+const player = new Player(prompt('what is your name?'));
+function Player(name){
+  this.name = player;
```

Comment:
name will most likely error trying to set `this.name = player` when the constructor is being used to define what `player` is in the first place. if it doesn't error, it will set `this.name` as undefined. we don't want either of those!
```suggestion
  this.name = name;
```